app/scripts/views/visits/item.js

```diff
@@ -16,9 +19,22 @@ define([
     },
 
     initialize: function () {
+      this.presenter = new VisitPresenter(this.model);
+
       this.listenTo(this.model, 'change', this.render);
     },
 
+    getContext: function () {
+      return this.presenter;
+    },
+
+    afterRender: function () {
+      // add size class
+      this.$el.addClass(this.presenter.getSizeClassName());
+      // add image position class
+      this.$el.find('.image').addClass(_.sample(['left', 'right']));
```

Comment:
Mildly related... I want the same items to always be on the left and right so that they don't reshift when you press the back button. I want to derive something from the visit/user_page so that it's consistent. I was thinking of doing `(title.length % 2) ? 'left' : 'right'`. Too dumb?